packages/playground/src/components/networks.vue

```diff
@@ -130,6 +131,10 @@ export default {
       default: () => null,
     },
     disabled: { type: Boolean },
+    isDiscourse: {
+      type: Boolean,
+      default: () => null,
+    },
```

Comment:
networks component shouldn't know which solution using it so we should expose `isDiscourse` variable instead we can expose something like `disableIpv4`, `disableIpv6`, ...etc so on